openfl-tutorials/interactive_api/Pytorch_Market_Re-ID/envoy_folder/market_shard_descriptor.py

```diff
@@ -103,13 +114,16 @@ def _check_before_run(self):
     @staticmethod
     def download():
         """Download Market1501 dataset."""
+        logger.info('Download Market1501 dataset.')
         if Path('Market-1501-v15.09.15').exists():
             return None
 
         output = 'Market-1501-v15.09.15.zip'
         if not Path(output).exists():
+            logger.info('Try to download.')
             url = 'https://drive.google.com/uc?id=0B8-rUzbwVRk0c054eEozWG9COHM'
```

Comment:
It could be variable in constructor or class variable in order to redefine it easily.

Reply:
It's another example from another PR. I have added `get_dataset` method here to demonstrate possible implementation without separating into ShardDescriptor and Dataset classes.